main.tex

```diff
@@ -156,15 +162,17 @@ \section*{Results}
 \subsection*{Functional variation}
```

Comment:
I've reworked the results section on functional variation. The main thing I've tried to deal with here is that, at this stage in the paper, we don't yet have any results on selection, and the results on selection (EHH analysis) was chopped from the Ag1000G phase 1 paper so we can't refer to it. And so if we try to predict a resistance phenotype at this point in the paper, we would have to take things out of order. To resolve this I have limited this section to reporting of population allele frequencies and patterns of LD. I thought we could then come back to the question of predicting resistance phenotype (and the nice table 2) later in the paper, after we have reported EHH results.